### ISSUE TITLE
Exception thrown with bb.edn

### ISSUE THREAD

**Issue by latacora-gabriel** [liquidz/antq#89]
Hi,
  Thanks for the handy library and github action! I encountered an exception with a bb.edn like the following:

```clj
{:tasks
 {:init (let [foo :foo] (assoc-in {} [:deps foo :bar] "baz"))}}
```
Running a clojure alias as [described in the readme](https://github.com/liquidz/antq#clojure-cli-tool) results in

```sh
$ clojure -M:outdated
Execution error (IllegalArgumentException) at antq.dep.clojure/extract-deps$fn (clojure.clj:41).
Don't know how to create ISeq from: clojure.lang.Symbol
```
If I rename :deps to :depz the error goes away.
A secondary issue I encountered was that adding `--skip babashka` didn't allow me to turn off bb. I ended up reverting to 0.12.4 so I could run antq.
Cheers

**Comment by liquidz** [comment]
@latacora-gabriel Thanks for your reporting!
I'll fix them.

**Comment by liquidz** [comment]
@latacora-gabriel Just released 0.15.3 #90

**Comment by latacora-gabriel** [comment]
Confirmed this fixed my issue. Thanks! 🎉
